api/lib/infrastructure/repositories/organization-repository.js

```diff
@@ -28,10 +28,10 @@ module.exports = {
 
   getByUserId(userId) {
     return Organization
-      .where({ userId: userId })
-      .fetchAll({ require: true })
-      .then((organizations)=>{
-        return organizations.models;
+      .where({ userId })
+      .fetchAll()
+      .then((organizations) => {
+        return (organizations) ? organizations.models : [];
```

Comment:
Il manque un test pour la valeur par défaut :/
Le test "should return an empty Array, when organization id is not found" ne passe pas rouge lorsque l'on change ça.